kumascript/macros/PreviousMenuNext.ejs

```diff
@@ -42,11 +42,11 @@ var s_Menu = mdn.localString({
 });
 
 if ($0) {
-    strPrevious = '<li><a class="button minimal" href="/' + lang + '/docs/' + string.replace($0, ' ', '_') + '">' + s_PreviousNext[0] + '</a></li>';
+    strPrevious = '<li><a class="button minimal" href="/' + lang + '/docs/' + $0.replaceAll(' ', '_') + '">' + s_PreviousNext[0] + '</a></li>';
```

Comment:
Wait, does this even work in 12? 
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll#browser_compatibility it was added in Node 15. 
I have Node 15 on my computer but our minimum is Node 12. 

The version of Node we use in CI is Node 12. If Node 12 doesn't support it, how come the tests didn't break?!

Reply:
Actually, what might have happened, in CI, is that the page is rendered but it has flaws (because of this), but that doesn't break the build. 

To be on the safe side, can we just replace (no pun intended) these with `$0.replace(/ /g, '_')`